src/Testcontainers/Builders/TestcontainersBuilderAzuriteExtension.cs

```diff
@@ -0,0 +1,139 @@
+      }
+
+      if (configuration.RunTableOnly || configuration.RunAllServices)
+      {
+        builder = builder.WithPortBinding(configuration.TablePort, AzuriteTestcontainerConfiguration.DefaultTablePort);
+      }
+
+      if (!string.IsNullOrWhiteSpace(configuration.Location))
+      {
+        builder = builder.WithBindMount(configuration.Location, AzuriteTestcontainerConfiguration.DefaultLocation);
+      }
+
+      builder = builder
+        .WithCommand(GetMainCommand(configuration))
+        .WithCommand(GetServiceEndpointArgs(configuration))
+        .WithCommand(GetWorkspaceLocation())
+        .WithCommand(GetSilentMode(configuration))
+        .WithCommand(GetLooseMode(configuration))
+        .WithCommand(GetSkipApiVersionCheck(configuration))
+        .WithCommand(GetDisableProductStyleUrl(configuration))
+        .WithCommand(GetDebugLog(configuration));
+
+      return builder;
+    }
+
+    private static string[] GetDebugLog(AzuriteTestcontainerConfiguration configuration)
+    {
+      var debugLogFilePath = Path.Combine(AzuriteTestcontainerConfiguration.DefaultLocation, "debug.log");
+      return configuration.DisableProductStyleUrl ? new[] { "--debug", debugLogFilePath } : null;
+    }
+
+    private static string GetDisableProductStyleUrl(AzuriteTestcontainerConfiguration configuration)
+    {
+      return configuration.DisableProductStyleUrl ? "--disableProductStyleUrl" : null;
+    }
+
+    private static string GetSkipApiVersionCheck(AzuriteTestcontainerConfiguration configuration)
+    {
+      return configuration.SkipApiVersionCheck ? "--skipApiVersionCheck" : null;
+    }
+
+    private static string GetLooseMode(AzuriteTestcontainerConfiguration configuration)
+    {
+      return configuration.LooseMode ? "--loose" : null;
+    }
+
+    private static string GetSilentMode(AzuriteTestcontainerConfiguration configuration)
+    {
+      return configuration.Silent ? "--silent" : null;
+    }
+
+    private static string[] GetWorkspaceLocation()
+    {
+      return new[] { "--location", AzuriteTestcontainerConfiguration.DefaultLocation };
+    }
+
+    private static string[] GetServiceEndpointArgs(AzuriteTestcontainerConfiguration configuration)
+    {
+      var args = new List<string>();
+
+      if (configuration.RunBlobOnly || configuration.RunAllServices)
+      {
+        args.Add("--blobHost");
+        args.Add(AzuriteTestcontainerConfiguration.DefaultBlobEndpoint);
+      }
+
+      if (configuration.RunQueueOnly || configuration.RunAllServices)
+      {
+        args.Add("--queueHost");
+        args.Add(AzuriteTestcontainerConfiguration.DefaultQueueEndpoint);
+      }
+
+      if (configuration.RunTableOnly || configuration.RunAllServices)
+      {
+        args.Add("--tableHost");
+        args.Add(AzuriteTestcontainerConfiguration.DefaultTableEndpoint);
+      }
+
+      return args.Count > 0 ? args.ToArray() : null;
```

Comment:
I think you can just return `args.ToArray()`.

Reply:
Fixed in [bddb8be](https://github.com/testcontainers/testcontainers-dotnet/pull/555/commits/bddb8be4dc91b51a89a704799bbe006d784e4f8c)